### ISSUE TITLE
Is the pruned_loss proportional to the max duration？

### ISSUE THREAD

**Issue by l2009312042** [k2-fsa/icefall#1309]
i wat to  compare loss between different experiments by changing  max_duration.
for example, is  the loss of  max duration =600 twice as much as 300? 
In the code, i see the following statement:
 pruned_loss = k2.rnnt_loss_pruned(...,reduction="sum", )

**Comment by csukuangfj** [comment]
We will divide pruned_loss by the number of frames later.

Please see
https://github.com/k2-fsa/icefall/blob/1ef349d120acef5d48feee58c4462a56f4a8c995/icefall/utils.py#L1176

**Comment by l2009312042** [comment]
thanks，so currently, the loss is increasing as max duration increases?

**Comment by csukuangfj** [comment]
no, loss per frame doesn't depend on max duration.

Sum of loss over all frames in a batch does depend on max duration.
